src/View/ViewContainer.vala

```diff
@@ -33,13 +33,13 @@ namespace Files.View {
                 return _window;
             }
 
-            set {
+            set construct {
```

Comment:
Should this be `set construct` or just `construct`? If it's settable I think we need to disconnect signals from the older object before connecting to the new one right?

Reply:
Dragging a tab to a new or different window causes this property to be set to a different window.  The signals are disconnected in line 38?

Reply:
On further investigation it turns out there is a pre-existing issue around changing the window associated with a viewcontainer.  This only occurs when dragging and dropping between windows because the viewcontainer is not recreated (it is created new when the new window or open-in actions are used).  I have fixed it here while there is attention on it but it could be done in a separate PR if desired.